platformio-example.ini

```diff
@@ -25,6 +25,7 @@ framework = arduino
 monitor_speed=115200
 lib_deps=
   U8g2
+  Adafruit BME680 Library
```

Comment:
Please also include the library in the Readme

Reply:
maybe the readme could just say "please install all libraries you see in platformio-example.ini lib_deps"?

Reply:
Ah, that's a good idea.   ".. and always use the newest versions."

Reply:
Will do